discover/portal/articles/210-setting-up/02-instance-settings/01-virtual-instances.markdown

```diff
@@ -0,0 +1,58 @@
+# Virtual Instances
+
+Here's a quick scenario: you already have a server hosting a site created with
+Liferay. That means you already have a hosted Liferay database. Let's say you
+need to set up another site that runs on Liferay. While there's more than one
+way to skin this cat, if you want to kill two birds with one stone, just create
+an additional Virtual Instance of Liferay. You'll get to have two separate sites
+(the birds), with a shared server and database (the stone). If you don't mind
+that the sites have a shared database and server, you can set up a
+separate Virtual Instance in two shakes of a lambs tail. 
```

Comment:
I don't think this is a great example, because if you just want a second site, then there are much simpler ways of doing that. 

Virtual instances should only be used for more complex environments. For example, if there is an installation of Liferay with several sites and users, roles, potentially other things (user groups, ...) defined and you want to do something completely separate, with different users. But you want to reuse the same installation (including any customizations you may have built for it). That's when you do several virtual instances.

The moment you want separate customizations or upgrade plans, that's when it's better to move to separate real instances.

Reply:
Hmm, I've never known a lot about Virtual Instances. What happens with OSGi config files and System Settings. All System configs apply to all virtual instances, right? What happens with search?

Reply:
Also, when you create a new virtual instance, does it inherit the current instance scoped configurations from the existing virtual instance, or is everything just the defaults?

Reply:
See here for edits: https://github.com/rbohl/liferay-docs/pull/80/commits/636625c3ea93a6a6e143030185a0277280aef7a6

I wouldn't be surprised if it needs more though.

Reply:
All system level scope configuration (everything in System Settings) is shared across all virtual instances. When a virtual instance is created, it does not get anything from the default instance.

I'll ready the edits and will answer there.